### ISSUE TITLE
Dataexplorer sends bad influxql

### ISSUE THREAD

**Issue by goller** [influxdata/chronograf#1004]
```
DEBU[12740] query                                         command=SHOW TAG VALUES FROM "consul_health_checks" WITH KEY IN () component=proxy db=nonstandardtelegraf host=chronograf.influxcloud.net:8086 rp=someotherrp

ERRO[12740] Error message received status code 400 from server: err: error parsing query: found ), expected identifier at line 1, char 58  component=server http_status =400
```

**Comment by goller** [comment]
somehow this happened to me again... here is the output on the JS console

![image](https://cloud.githubusercontent.com/assets/1922921/23913570/6c10e2bc-08b1-11e7-92a8-b7d0e05653b1.png)

**Comment by nhaugo** [comment]
Cannot reproduce

**Comment by goller** [comment]
Ok I now know how to reproduce this:

1. Create a new non-default retention policy for a database like `telegraf`
2. Select the db and new rp from databases.
3. Select measurement

**Comment by goller** [comment]
Here is the line of code: https://github.com/influxdata/chronograf/blob/master/ui/src/data_explorer/components/TagList.js#L47

**Comment by goller** [comment]
This was fixed quite a long time ago in my pr #1182
